Source/WebGPU/WebGPU/CommandEncoder.h

```diff
@@ -73,6 +73,12 @@ class CommandEncoder : public CommandsMixin, public RefCountedAndCanMakeWeakPtr<
     {
         return adoptRef(*new CommandEncoder(device));
     }
+#if ENABLE(WEBGPU_SWIFT)
+    inline Ref<CommandBuffer> createCommandBuffer(id<MTLCommandBuffer> commandBuffer, Device& device, id<MTLSharedEvent> sharedEvent, uint64_t sharedEventSignalValue)
```

Comment:
This change seems out of place with the others, at the least I think this can be a private member function right? It is called from CommandEncoder logic in Swift

Reply:
That was because of stacked PRs. It's no longer in this PR.